src/test/java/de/retest/recheck/ui/diff/DurationDifferenceTest.java

```diff
@@ -0,0 +1,46 @@
+package de.retest.recheck.ui.diff;
+
+import static de.retest.recheck.XmlTransformerUtil.toXmlFragmentViaJAXB;
+import static de.retest.recheck.util.ApprovalsUtil.verifyXml;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class DurationDifferenceTest {
+
+	@Test
+	public void same_duration() {
+		final DurationDifference difference = DurationDifference.differenceFor( 70, 70 );
+		assertThat( difference ).isNull();
+	}
+
+	@Test
+	public void expected_is_longer_but_below_threshold() {
+		final DurationDifference difference = DurationDifference.differenceFor( 1000, 1 );
+		assertThat( difference ).isNull();
+	}
+
+	@Test
+	public void expected_is_longer_and_above_threshold() {
+		final DurationDifference difference = DurationDifference.differenceFor( 1001, 1 );
+		assertThat( difference ).isNull();
```

Comment:
Isn't this a bug? A difference should have been produced here. as the threshold is above. (Use `Math.abs` here?)

Reply:
if the expected value is greater than the actual value, the duration difference should be negative which is kind of always below the threshold isn't it?